src/secret_sharing/malicious_replicated.rs

```diff
@@ -41,12 +40,6 @@ impl<F: Field> MaliciousReplicated<F> {
     pub fn rx(&self) -> Replicated<F> {
         self.rx
     }
-
-    /// Returns a pair of replicated secret sharings. One of "one", one of "r"
-    #[allow(dead_code)]
-    pub fn one(helper_role: Role, r_share: Replicated<F>) -> Self {
-        Self::new(Replicated::one(helper_role), r_share)
-    }
```

Comment:
This is still definitely necessary. Don't remove this. We just need the ProtocolContext to invoke this, because it should know what `r_share` is.